include/bredis/Error.hpp

```diff
@@ -20,13 +20,14 @@ class bredis_category : public boost::system::error_category {
 };
 
 class Error {
-    static const bredis_category &category;
+    static const bredis_category category;
 
   public:
+    Error() {};
     static inline boost::system::error_code make_error_code(bredis_errors e);
 };
 
-const bredis_category &Error::category = bredis_category();
+const bredis_category Error::category{};
```

Comment:
Hm, I believe this will cause duplicate definition errors. Consider this style
```
inline
bredis_category const&
get_error_category()
{
    static bredis_category const cat{};
    return cat;
}
```